### PR TITLE
[stable/prestashop] Add country and language env-vars

### PR DESCRIPTION
**Pull request by andresbono**
Signed-off-by: Andrés Bono <abjimenez@bitnami.com>

#### What this PR does / why we need it:

Adds country and language env-vars

#### Which issue this PR fixes
  - fixes https://github.com/bitnami/bitnami-docker-prestashop/issues/70

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by juan131 — APPROVED** [review]
Great! Thanks for the contribution!

File: stable/prestashop/values.yaml [hunk 1]
```diff
@@ -59,6 +59,16 @@ prestashopLastName: User
 ##
 prestashopCookieCheckIP: "no"
 
+## Country
+## ref: https://github.com/bitnami/bitnami-docker-prestashop#configuration
+##
+prestashopCountry: "en"
```
Comment (juan131) [inline stable/prestashop/values.yaml]: This value is wrong.

**Review by juan131 — CHANGES_REQUESTED** [review]

**Comment by juan131** [comment]
You need to sign every commit

File: stable/prestashop/values.yaml (hunk 1 above)
Comment (andresbono) [reply stable/prestashop/values.yaml]: Changed

**Pull request merged**